webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -105,7 +105,18 @@ export default class CreateIssueModal extends PureComponent {
         this.setState(initialState, this.props.close);
     };
 
-    handleRepoChange = (repo) => this.setState({repo});
+    handleRepoChange = (repo) => {
```

Comment:
I updated this since I noticed that on switching repos previously selected labels didn't reset.